### PR TITLE
Master

### PR DESCRIPTION
**Pull request by bgarneau**
New blogpost

### CODE REVIEW

File: _posts/2015-03-20browserVSDIP.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+While the two first tests successfully displayed the image, the two last one didn’t show anything, it was like the line was not even in the html file.
+By opening the browser’s console, I saw, for both failed attempts, an error message saying “Not allowed to load local resource”.
+Interesting fact, the console was the only place where I could see any indication that there was even an attempt to load the local JPEG image. The web page did not show the classic “file unavailable” icon, there was no dialog box, and no mention on the browser’s status bar, like you could expect to see when a pop-up page is blocked.
+Unlike for pop-up pages, the browser won’t offer you to allow access to local files, it will just ignore the instruction.
+
+But why?
+
+I had to call a developer to find out more about this…
+
+![A photo of me, calling a developer]({{ site.baseurl }}/images/2.jpg)
```
Comment (pastjean) [inline _posts/2015-03-20browserVSDIP.md]: c'est présentement un mess dans les images, on va essayer de mettre ca mieu.

images url 1-2-3 ... créé toi un dossier pour ton post images/2015-02-01/1.jpg ou renomme les avec la date de ton post ie :images/2015-02-01_1.jpg

File: _posts/2015-03-20browserVSDIP.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+title: "Browser strangelove"
+#subtitle: "Or how I learned to stop worrying and love the Desktop Integration Package"
+
+author:
+  name: Benoit Garneau
+  bio: Product Specialist
+  image: megaman.jpg
+---
+
+![Post main image]({{ site.baseurl }}/images/1.jpg)
```
Comment (pastjean) [inline _posts/2015-03-20browserVSDIP.md]: same as 2.jpg

File: _posts/2015-03-20browserVSDIP.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+But why?
+
+I had to call a developer to find out more about this…
+
+![A photo of me, calling a developer]({{ site.baseurl }}/images/2.jpg)
+ 
+The developer confirmed that opening a local file from the DIP is performed in a different way. "javascript:window.external.OpenDesktopConnectorUri()" is a javascript call from the Internet Explorer browser embedded in the DIP to a .NET method implemented in the DIP itself. This mechanism is not possible from a standalone browser. This call to the DIP is for working around security restrictions imposed by browsers. Those security restrictions differ from one browser to another (e.g. Chrome vs. IE), and even differ from one version of IE to another, the compatibility view settings, etc.
+I later found out that you could bypass the browser’s security by completely disabling the Internet Explorer Local Machine Zone Lockdown, which is like tearing down a whole wall of your house to get all the groceries inside in one trip, but that’s another story.
+
+![Slim Pickins riding a falling bomb with his classic IE logo]({{ site.baseurl }}/images/3.jpg)
```
Comment (pastjean) [inline _posts/2015-03-20browserVSDIP.md]: same as 2.jpg

File: _posts/2015-03-20browserVSDIP.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+#subtitle: "Or how I learned to stop worrying and love the Desktop Integration Package"
+
+author:
+  name: Benoit Garneau
+  bio: Product Specialist
+  image: megaman.jpg
+---
+
+![Post main image]({{ site.baseurl }}/images/1.jpg)
+Not too long ago, a customer opened a case about the troubles he had loading an Office file from a search results page. The said file was on his workstation. From his browser, he clicked on the open link to access the local file, only to get an error message.
```
Comment (pastjean) [inline _posts/2015-03-20browserVSDIP.md]: The said file ;) ... le dit fichier ... on peut pas traduire ça comme ça. -> "The file was hosted locally on his workstation"

... he clicked on the open link to access the local file... ->  ... he clicked on the link to access the file, ...

**Comment by bgarneau** [comment]
Did a few changes... Should be better now.

File: _posts/2015-03-20browserVSDIP.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+Curious about this issue, I ran a few tests on a virtual machine, with indexed local files, just like the customer did on his side before opening the case.
```
Comment (pastjean) [inline _posts/2015-03-20browserVSDIP.md]: ... , with indexed local files ... -> "enlever la virgule"
... with indexed local files -> with locally indexed file

File: _posts/2015-03-20browserVSDIP.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+Test one
+Typing File:///C:/test/test.jpg on the address bar of IE11.
+Test two
+<img src=File:///C:\test\test.jpg /> in an html file loaded locally on the machine.
+Test three
+<img src=File:///C:\test\test.jpg /> in an html file hosted on a distant server.
+Test four
+<img src=File:///C:\test\test.jpg /> in an html file on a network shared folder.
+
+While the two first tests successfully displayed the image, the two last one didn’t show anything, it was like the line was not even in the html file.
```
Comment (pastjean) [inline _posts/2015-03-20browserVSDIP.md]: the two last one -> the last two ones

File: _posts/2015-03-20browserVSDIP.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+By opening the browser’s console, I saw, for both failed attempts, an error message saying “Not allowed to load local resource”.
+Interesting fact, the console was the only place where I could see any indication that there was even an attempt to load the local JPEG image. The web page did not show the classic “file unavailable” icon, there was no dialog box, and no mention on the browser’s status bar, like you could expect to see when a pop-up page is blocked.
+Unlike for pop-up pages, the browser won’t offer you to allow access to local files, it will just ignore the instruction.
```
Comment (pastjean) [inline _posts/2015-03-20browserVSDIP.md]: ... the browser won’t offer you to allow access to local files ... -> the browser won’t allow access to local files.

File: _posts/2015-03-20browserVSDIP.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+But why?
+
+I had to call a developer to find out more about this…
+
+![A photo of me, calling a developer]({{ site.baseurl }}/images/2.jpg)
+ 
+The developer confirmed that opening a local file from the DIP is performed in a different way. "javascript:window.external.OpenDesktopConnectorUri()" is a javascript call from the Internet Explorer browser embedded in the DIP to a .NET method implemented in the DIP itself. This mechanism is not possible from a standalone browser. This call to the DIP is for working around security restrictions imposed by browsers. Those security restrictions differ from one browser to another (e.g. Chrome vs. IE), and even differ from one version of IE to another, the compatibility view settings, etc.
+I later found out that you could bypass the browser’s security by completely disabling the Internet Explorer Local Machine Zone Lockdown, which is like tearing down a whole wall of your house to get all the groceries inside in one trip, but that’s another story.
```
Comment (pastjean) [inline _posts/2015-03-20browserVSDIP.md]: nice one :dart:

File: _posts/2015-03-20browserVSDIP.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+![Slim Pickins riding a falling bomb with his classic IE logo]({{ site.baseurl }}/images/3.jpg)
+ 
+Let’s just say that in the end, if you want to make sure that all your files can be opened through a web browser without sacrificing security, you can host your documents on a cloud based solution.
```
Comment (pastjean) [inline _posts/2015-03-20browserVSDIP.md]: Let’s just say that in the end, -> Let’s just say that, in the end,    `comma,virgule`

**Pull request merged**